profile/README.md

```diff
@@ -0,0 +1,25 @@
+# Puroto
+
+> We are building a future social network for furries.
+
+<br>
+
+Let's go! Here are some useful links to get you started:
+
+- [Puroto](https://puroto.net) - the project homepage
+- [Discord](https://discord.puroto.net) - Join our server
+- [Blog](https://blog.puroto.net) - Get the latest news on Puroto
+- [Twitter](https://twitter.com/PurotoApp) - Follow us for more info
+
+## Be a part of the project
+
+All of our projects are following the guidelines you can find in our 
+[Code of Conduct](https://github.com/.github/CODE_OF_CONDUCT.md) and are
+under the [GPLv3 License](https://github.com/.github/LICENSE.md).
+
+Please consider also reading our [Contributing](https://github.com/.github/CONTRIBUTING.md) 
+and our [Security](https://github.com/.github/SECURITY.md) policies.
```

Comment:
The same as the comment on the security disclosure.

Reply:
If someone want to contribute, they can find the useful markdown right away on puroto organisiation homepage

Reply:
I'll move that to #4.